tasks/service-runner/app/workers/pull-worker.ts

```diff
@@ -12,6 +12,9 @@ const activeSocketKeepAlive = 6_000;
 const maxSockets = 1;
 const maxFreeSockets = 1;
 const maxRetries = 3;
+let pullCounter = 0;
+// how many pulls to execute before logging - used to keep log message count reasonable
+const pullLogPeriod = 10;
```

Comment:
😬  - this is going to add logging every 5 seconds when there is no work. I'm going to miss the nice clean logs that only logged messages when there were requests going on.

Reply:
We could add another env var 😬